specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -32,7 +32,6 @@ options:
     model-namespace: false
   "@azure-tools/typespec-ts":
     emitter-output-dir: "{output-dir}/{service-dir}/contoso-widgetmanager"
-    is-modular-library: true
```

Comment:
@maririos and @mikeharder Could you help review and approve this pr? 

JS has adjusted the default rule for option `is-modular-library` and this setting won't be necessary in our contoso project.

Reply:
Should the setting also be removed from our `widget` sample?

https://github.com/Azure/azure-rest-api-specs/blob/c3c93e9102521197fffb02498222aa1c2375d69a/specification/widget/data-plane/WidgetAnalytics/tspconfig.yaml#L31

Also, I see a lot more hits for `is-modular-library`, both `true` and `false`, in existing specs in main.  Why does this PR only change these 3?